artemis-ra/src/main/java/org/apache/activemq/artemis/ra/referenceable/SerializableObjectRefAddr.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.activemq.artemis.jms.referenceable;
```

Comment:
do you really need this move?

I'm thinking about Wildfly integration here.

Reply:
no we didn't, but its only applicable to RA which has its own module we can make that package in the RA module.

Reply:
Ill add back like you mention in the other comment a back compatibility extension class in the old location.